### ISSUE TITLE
Multimedia keys not working + Can't see wrapper version

### ISSUE THREAD

**Issue by jorgerc85** [headsetapp/headset-electron#151]
Hi, I searched through the issues and found that multimedia keys should be working but for some reason they don't on my machine. Multimedia keys work normally on other apps. 

I am running Ubuntu 16.04 and latest version of Headset available on github. (Core: 1.8.2, Wrapper: ) This is the other issue. For some reason I can't see the wrapper version. Could this be at all related to my issue?
![image](https://user-images.githubusercontent.com/8603444/38693142-7117494a-3e5c-11e8-9983-4acb1398abc6.png)

Is there a way to debug this issue from my machine? Thanks!

**Comment by fcastilloec** [comment]
Hi, I'm going to need some more info from you to be able to help. First, I think that the Wrapper issue might be easy to fix. This could have happened if the app was open when an update was issued, and some old cache files stayed in the system. Can you please delete the folder `$HOME/.config/Headset`. This will reset your app and will ask you to sign in. You won't lose any of your Collections/Likes, as soon as you sign in, everything will sync back.

After doing the previous step, you should also try the multimedia keys. If these are still not working, can you please start Headset from the terminal with this command: `DEBUG=headset:mediaKeys headset` Then try to play a song and use the multimedia keys to control it, please paste the output here.

**Comment by jorgerc85** [comment]
Thanks for the fast reply. The version issue was fixed after deleting the config folder (My current versions: Core: 1.8.2, Wrapper: v1.8.1). 

Multimedia keys still don't work though. The output from running the command:
```
$ DEBUG=headset:mediaKeys headset
  headset:mediaKeys Registering media Keys +0ms
  headset:mediaKeys Grabbed media keys for 'gnome' +60ms
Failed to encode dictionary
Failed to encode reply value
Failed to encode dictionary
Failed to encode reply value

```

**Comment by fcastilloec** [comment]
@jorgerc85 I'm going to need to know more details. Can you please start headset with `DEBUG=headset* headset` Start playing one song, and then try to pause it using the media keys. Don't press other media keys, just trying to pause it. Feel free to press it a couple of times if necessary. Then paste the output here.

**Comment by jorgerc85** [comment]
Here you go. I pressed pause for a while and nothing happened. I paused the song manually at the very end.
```
$ DEBUG=headset* headset
  headset Running as developer: false +0ms
  headset Starting Headset +165ms
  headset Main window finished loading +2s
  headset Initializing MPRIS and registering MediaKeys +22ms
  headset:mediaKeys Registering media Keys +0ms
  headset:mediaKeys Grabbed media keys for 'gnome' +42ms
Failed to encode dictionary
Failed to encode reply value
Failed to encode dictionary
Failed to encode reply value
  headset Player window finished loading +3s
  headset:player2Win [ 'onReady', null ] +0ms
  headset:win2Player [ 'setVolume', 75 ] +0ms
  headset:mpris Playback status: 'Stopped' +0ms
  headset:win2Player [ 'unMute', null ] +2ms
  headset:mpris Playback status: 'Stopped' +1ms
  headset:win2Player [ 'shuffle', true ] +2m
  headset:mpris Playback status: 'Stopped' +2m
  headset:win2Player [ 'loadVideoById', 'Stl9yl9lgWQ' ] +6ms
  headset:mpris Playback status: 'Stopped' +5ms
  headset:win2Player [ 'trackInfo',
  headset:win2Player   { albumTrackList: null,
  headset:win2Player     artist: 'The Moon and the Nightspirit',
  headset:win2Player     duration: 289,
  headset:win2Player     durationClockFormat: '4:49',
  headset:win2Player     id: 'Stl9yl9lgWQ',
  headset:win2Player     image: 'https://i.ytimg.com/vi/Stl9yl9lgWQ/hqdefault.jpg',
  headset:win2Player     pin: null,
  headset:win2Player     thumbnail: 'https://i.ytimg.com/vi/Stl9yl9lgWQ/mqdefault.jpg',
  headset:win2Player     title: 'Égi Táltos',
  headset:win2Player     ts: 1523556158,
  headset:win2Player     type: 'video' } ] +1ms
  headset:mpris [ 'Track Info:',
  headset:mpris   { 'xesam:artist': [ 'The Moon and the Nightspirit' ],
  headset:mpris     'xesam:title': 'Égi Táltos',
  headset:mpris     'xesam:url': 'https://www.youtube.com/watch?v=Stl9yl9lgWQ',
  headset:mpris     'mpris:artUrl': 'https://i.ytimg.com/vi/Stl9yl9lgWQ/mqdefault.jpg',
  headset:mpris     'mpris:length': 289000000 } ] +4ms
  headset:mpris Playback status: 'Stopped' +2ms
  headset:player2Win [ 'onStateChange', -1 ] +2m
  headset:player2Win [ 'onStateChange', 3 ] +2ms
  headset:win2Player [ 'playVideo', null ] +341ms
  headset:mpris Playback status: 'Playing' +337ms
  headset:player2Win [ 'onStateChange', 1 ] +2s
  headset:player2Win [ 'currentTime', 0.2245401220703125 ] +221ms
  headset:player2Win [ 'currentTime', 0.6332440400543213 ] +499ms
  headset:player2Win [ 'currentTime', 1.1339110400543213 ] +500ms
  headset:player2Win [ 'currentTime', 1.6344749465942383 ] +500ms
  headset:player2Win [ 'currentTime', 2.1344090400543214 ] +500ms
  headset:player2Win [ 'currentTime', 2.6344631125335694 ] +500ms
  headset:player2Win [ 'currentTime', 3.1352700400543214 ] +501ms
  headset:player2Win [ 'currentTime', 3.6329228741149904 ] +501ms
  headset:player2Win [ 'currentTime', 4.135532967575074 ] +502ms
  headset:player2Win [ 'currentTime', 4.633254112533569 ] +496ms
  headset:player2Win [ 'currentTime', 5.134264040054322 ] +500ms
  headset:player2Win [ 'currentTime', 5.632482946594238 ] +500ms
  headset:player2Win [ 'currentTime', 6.134065040054321 ] +501ms
  headset:player2Win [ 'currentTime', 6.6317039465942385 ] +499ms
  headset:player2Win [ 'currentTime', 7.130113040054321 ] +500ms
  headset:player2Win [ 'currentTime', 7.630545925613403 ] +500ms
  headset:player2Win [ 'currentTime', 8.133414112533568 ] +500ms
  headset:player2Win [ 'currentTime', 8.63308904005432 ] +500ms
  headset:player2Win [ 'currentTime', 9.132134112533569 ] +500ms
  headset:player2Win [ 'currentTime', 9.633230040054322 ] +500ms
  headset:player2Win [ 'currentTime', 10.132343112533569 ] +501ms
  headset:player2Win [ 'currentTime', 10.63238911253357 ] +499ms
  headset:player2Win [ 'currentTime', 11.132557112533568 ] +500ms
  headset:player2Win [ 'currentTime', 11.634297019073486 ] +500ms
  headset:player2Win [ 'currentTime', 12.133840040054322 ] +501ms
  headset:player2Win [ 'currentTime', 12.634000040054321 ] +499ms
  headset:player2Win [ 'currentTime', 13.134222040054322 ] +500ms
  headset:player2Win [ 'currentTime', 13.631942946594238 ] +500ms
  headset:player2Win [ 'currentTime', 14.13362804005432 ] +500ms
  headset:player2Win [ 'currentTime', 14.63261911253357 ] +501ms
  headset:player2Win [ 'currentTime', 15.133622040054322 ] +499ms
  headset:player2Win [ 'currentTime', 15.633550040054322 ] +501ms
  headset:player2Win [ 'currentTime', 16.133635040054322 ] +500ms
  headset:player2Win [ 'currentTime', 16.63250911253357 ] +500ms
  headset:player2Win [ 'currentTime', 17.133676040054322 ] +499ms
  headset:player2Win [ 'currentTime', 17.635575133514404 ] +503ms
  headset:player2Win [ 'currentTime', 18.13348204005432 ] +497ms
  headset:player2Win [ 'currentTime', 18.6303499256134 ] +501ms
  headset:player2Win [ 'currentTime', 19.13346804005432 ] +499ms
  headset:player2Win [ 'currentTime', 19.633580040054323 ] +500ms
  headset:player2Win [ 'currentTime', 20.13373304005432 ] +500ms
  headset:player2Win [ 'currentTime', 20.63372004005432 ] +500ms
  headset:player2Win [ 'currentTime', 21.13376504005432 ] +500ms
  headset:player2Win [ 'currentTime', 21.63356404005432 ] +501ms
  headset:player2Win [ 'currentTime', 22.13359504005432 ] +499ms
  headset:player2Win [ 'currentTime', 22.631015091552733 ] +501ms
  headset:player2Win [ 'currentTime', 23.13641304005432 ] +499ms
  headset:player2Win [ 'currentTime', 23.63714504005432 ] +500ms
  headset:player2Win [ 'currentTime', 24.13645104005432 ] +501ms
  headset:player2Win [ 'currentTime', 24.633486040054322 ] +500ms
  headset:player2Win [ 'currentTime', 25.13252611253357 ] +499ms
  headset:player2Win [ 'currentTime', 25.633232019073485 ] +500ms
  headset:player2Win [ 'currentTime', 26.134713040054322 ] +501ms
  headset:player2Win [ 'currentTime', 26.63498204005432 ] +500ms
  headset:win2Player [ 'pauseVideo', null ] +29s
  headset:mpris Playback status: 'Paused' +29s
  headset:player2Win [ 'onStateChange', 2 ] +394ms
```

**Comment by jorgerc85** [comment]
I think I found the cause of this issue. It's not related to Headset.

The machine I'm using has custom assigned multimedia keys. For example:
"Play/Pause" binds to this command:
`dbus-send --print-reply --dest=org.mpris.MediaPlayer2.spotify /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.PlayPause`

I'm sorry I wasted your time trying to find a solution for this issue. I'll try to get this working using a similar approach. Thanks!
